client/disputes/utils.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { CachedDispute } from 'wcpay/types/disputes';
+import { formatCurrency } from 'wcpay/utils/currency';
+
+interface GetDisputesNoticeStringArgs {
+	/**
+	 * Active disputes (awaiting a response) to generate the notice string for.
+	 */
+	activeDisputes: CachedDispute[];
+}
```

Comment:
Is this a recommended pattern – defining a type for one function's args? Is it impractical to do this inline, since this "type" is tightly coupled to the function?

Alternatively, is this a generally useful struct type – if so, could rename it.

Reply:
Agree this is a bit heavy for a single function arg. I have no preference for a pattern here.

I've changed this to a single arg function in in 335587d.